Sources/Carting/Commands/Options.swift

```diff
@@ -14,7 +14,12 @@ struct Options: ParsableArguments {
     @Option(name: [.short, .long], default: ProcessInfo.processInfo.environment["PROJECT_DIR"], help: "The project directory path.")
     var path: String?
 
-    @Option(name: [.short, .long], default: .list, help: "Format of input/output file paths: file - using simple paths, list - using xcfilelists")
+    @Argument(help: "The project directories that contains frameworks to proceed")
```

Comment:
It seems like this description has already been used in `Info.swift` as well as previous `@Option`'s help text. Let's improve this and use some constants like we do with AppConfiguration?

Reply:
[String] will be simplified to String. Running info for multiple directories doesn't seem to be useful

Reply:
Info will have different @Argument (just String) and they description will differ

Reply:
Also, this argument here will recieve default value